src/components/class/Announcement.tsx

```
@@ -1,148 +1,152 @@
+                  >
+                    <HiCheck className="h-4 w-4" />
+                    <span>Save Changes</span>
+                  </Button.Primary>
+                  <Button.Light onClick={() => setEditing(false)}>
+                    Cancel
+                  </Button.Light>
+                </div>
+              </div>
+            ) : (
+              <div className="richText">
+                <div dangerouslySetInnerHTML={{ __html: remarks }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)